optaplanner-docs/src/modules/ROOT/pages/release-notes/release-notes-8.adoc

```diff
@@ -1,6 +1,8 @@
 [[releaseNotes-8.x]]
 == OptaPlanner 8.x Release Notes
 
+////
+TODO: enable after the 8.29.x branch is created.
```

Comment:
I don't think this is necessary. The docs will only be published with 8.29, so we can commit freely.

That said, with my latest commit, I think that this is now a conflict.